lesson_2/calculator.py

```diff
@@ -0,0 +1,35 @@
+# calculator program that will:
+    # Ask the user for two numbers.
+    # Ask the user for operation to perform: add, subtract, multiply or divide.
+    # Perform the calculation and display the results
+
+print('Welcome to Calculator!')
+
+# no error for input/ will crash if input is wrong --> would add error if was function
+num1 = int(input('Enter a number: '))
+num2 = int(input('Enter another number: '))
+
+operation = input('What operation would you like to perform?\n'
+                  '1) Addition 2) Subtraction 3) Multiplication 4) Division: ')
```

Comment:
question (non-blocking): is using numbers for the operation a course requirement?

If not, I'd use the operators themselves (i.e. `+`, `-`, `*`, and `/`). The reason for this is because it would make the match block you have below more readable (I don't need to read the comments to know what each case represents because for example, `case '+'` instantly tells my brain that's the addition case.

Reply:
so valid, it wasn't a requirement but they used it as an example so I followed their way. Agree that using operator as input would be cleaner!